docs/wiki/framework/cargo-framework.md

```diff
@@ -50,6 +50,7 @@ Event Name | Passed Parameter(s) | Locality | Description
 ---------- | ----------- | ------------------- | --------
 `ace_cargoLoaded` | [_item, _vehicle] | Global | Cargo has been Loaded into vehicle
 `ace_cargoUnloaded` | [_item, _vehicle] | Global | Cargo has been Unloaded from vehicle
+`ace_cargoRemoved` | [_itemClass, _vehicle, _amount, _itemsRemoved] | Global | Cargo has been removed (deleted) from vehicle. `_amount` is the requested number of items, `_itemsRemoved` is the actual amount that was removed.
```

Comment:
This column looks terrible with that much text. Is there any other way I could deliver this info without making it cumbersome? @bux

Reply:
```
`ace_cargoRemoved` | [_itemClass, _vehicle, _amountRequested, _itemsRemoved] | Global | Cargo has been removed (deleted) from vehicle
```
Simply changed argument to `_amountRequested` and I think it's all self-explanatory like that.